src/app/recipe-list/recipe-list.component.html

```diff
@@ -0,0 +1,10 @@
+<div *ngIf="recipeList$ | async as recipes">
```

Comment:
We may need to adapt styling to match the site. Haven't viewed it so I may be wrong.

Reply:
I can do that on my branch after we merge this and I rebase from develop.